packages/design-system-react/src/types/index.ts

```diff
@@ -142,6 +142,119 @@ export type BadgeWrapperCustomPosition = {
   left?: number | string | undefined;
 };
 
+/**
+ * Box - all spacing-related props
+ */
+export type BoxSpacing = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
+/**
+ * Box - flexDirection
+ */
+export enum BoxFlexDirection {
+  Row = 'flex-row',
+  RowReverse = 'flex-row-reverse',
+  Column = 'flex-col',
+  ColumnReverse = 'flex-col-reverse',
+}
+
+/**
+ * Box - flexWrap
+ */
+export enum BoxFlexWrap {
+  NoWrap = 'flex-nowrap',
+  Wrap = 'flex-wrap',
+  WrapReverse = 'flex-wrap-reverse',
+}
+
+/**
+ * Box - alignItems
+ */
+export enum BoxAlignItems {
+  Start = 'items-start',
+  Center = 'items-center',
+  End = 'items-end',
+  Stretch = 'items-stretch',
+  Baseline = 'items-baseline',
+}
+
+/**
+ * Box - justifyContent
+ */
+export enum BoxJustifyContent {
+  Start = 'justify-start',
+  Center = 'justify-center',
+  End = 'justify-end',
+  Between = 'justify-between',
+  Around = 'justify-around',
+  Evenly = 'justify-evenly',
+}
+
+/**
+ * Box - borderRadius
+ */
+export enum BoxBorderRadius {
+  None = 'rounded-none',
+  Sm = 'rounded-sm', // 4px
+  Md = 'rounded-md', // 6px
+  Lg = 'rounded-lg', // 8px
+  Full = 'rounded-full',
+}
+
+/**
+ * Box - borderWidth
+ */
+export enum BoxBorderWidth {
+  None = 'border-0',
+  Sm = 'border',
+  Md = 'border-2',
+  Lg = 'border-4',
+  Xl = 'border-8',
+}
```

Comment:
Border Width on extension is numerical not sure if we want to align here?
https://github.com/MetaMask/metamask-extension/blob/main/ui/components/component-library/box/box.types.ts#L349-L354